scripts/haas_network.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=HaaS Network Server 
+After=network.target
```

Comment:
So again, `After=` doesn't actually introduce a hard dependency on postgresql; Unless you do `systemctl enable postgresql` at some point, it won't have any effect. It only has an effect if postgres is otherwise set to start on that machine, in which case it makes sure the haas network daemon starts *after* postgres. If we're running the two on separate machines, then this won't do anything, since postgresql will be disabled on the machine running the haas network daemon.

I'm actually not 100% sure this matters; it may be the case that you can start things out of order and be fine. I would test a few things. First:

1. Start the haas network daemon, configured to use postgres, with postgres running.
2. Do an api call that puts a network action in the queue. This will *probably* result in some kind of error from the network daemon.
3. Start postgres.

You should also try the above without step (2).

It may be that things will just start working once postgres comes up, in which case it may be fine for them to start out of order.